tslib/src/azure/app-insights.ts

```diff
@@ -1,6 +1,6 @@
 import config from '../config'
 import { isNil } from 'ramda'
-const appInsights = require('applicationinsights')
+import * as appInsights from 'applicationinsights'
```

Comment:
Alarm bells are ringing here.  Please do test this.   Application Insights is very sensitive to ejs / cjs differences and although we are writing typescript esm, we are compiling to cjs to be run on node.